### DISCUSSION TITLE
How to Stop Discharging of Senec Storage Without Charging From the Grid?

### DISCUSSION THREAD

**Discussion by JeeperS1987** [nobl/ioBroker.senec#505] [Q&A]
Hello,

I would like to stop the discharging of my Senec storage. The storage has a capacity of only 7.5 kWh, which is not enough to get through the night. However, I want the storage to discharge only when the electricity price increases, and otherwise stop discharging when the electricity price is low.

I have found the data point "ForceLoadBattery," which stops the discharging but also charges the storage from the grid, which I do not want. Does anyone have an idea which data point could be used to stop only the discharging without charging the storage?

Thank you and regards.

**Comment by nobl** [comment]
In the past there was an option to prohibit charging / discharging of the senec via the webinterface.
This got removed and unless someone can confirm the original behaviour of these, I cannot include such an option :(

**Comment by nobl** [comment]
see: https://github.com/nobl/ioBroker.senec/discussions/504
